src/Core/Settings.h

```diff
@@ -593,6 +593,7 @@ class IColumn;
     M(Bool, input_format_null_as_default, true, "For text input formats initialize null fields with default values if data type of this field is not nullable", 0) \
     M(Bool, input_format_arrow_import_nested, false, "Allow to insert array of structs into Nested table in Arrow input format.", 0) \
     M(Bool, input_format_orc_import_nested, false, "Allow to insert array of structs into Nested table in ORC input format.", 0) \
+    M(Int64, row_batch_size, 100'000, "Batch size when reading ORC stripes.", 0) \
```

Comment:
Maybe it worth name it like other format related settings? And also add a note about which format supports it?

Reply:
right, my bad. Fixing